### PR TITLE
chapter14_part2: /110_Multi_Field_Search/05_Multiple_query_strings.asciidoc

### PR DESCRIPTION
**Pull request by richardwei2008**
初译

### CODE REVIEW

File: 110_Multi_Field_Search/05_Multiple_query_strings.asciidoc [hunk 1]
```diff
@@ -23,15 +19,9 @@ GET /_search
 --------------------------------------------------
 // SENSE: 110_Multi_Field_Search/05_Multiple_query_strings.json
 
-The `bool` query takes a _more-matches-is-better_ approach, so the score from
-each `match` clause will be added together to provide the final `_score` for
-each document. Documents that match both clauses will score higher than
-documents that match just one clause.
+`bool` 查询的采取 _more-matches-is-better_ 匹配越多越好的方式，所以每条 `match` 语句的评分结果会被加在一起，从而为每个文档提供最终的分数 `_score` 。能与两条语句同时匹配的文档比只与一条语句匹配的文档得分要高。
```
Comment (xuej) [inline 110_Multi_Field_Search/05_Multiple_query_strings.asciidoc]: `bool` 查询的采取=》去掉"的"？

**Comment by xuej** [comment]
LGTM

**Comment by medcl** [comment]
PR commits 有问题

**Comment by richardwei2008** [comment]
pls refer to new PR https://github.com/elasticsearch-cn/elasticsearch-definitive-guide/pull/356

**Pull request closed without merging**